bin/test.sh

```diff
@@ -61,6 +61,16 @@ function check_gpdb() {
     environment_compose exec gpdb su gpadmin -l -c "pg_isready"
 }
 
+function check_spark() {
+    environment_compose exec spark curl -f http://localhost:10213
+}
+
+function run_spark_tests() {
+    environment_compose exec spark beeline -u jdbc:hive2://localhost:10213 -e 'select 1;' &&
+        environment_compose exec spark beeline -u jdbc:hive2://localhost:10213 -e 'show databases;' &&
```

Comment:
nit: I would uppercase for compatibility with `run_hadoop_tests`.
```suggestion
    environment_compose exec spark beeline -u jdbc:hive2://localhost:10213 -e 'SELECT 1;' &&
        environment_compose exec spark beeline -u jdbc:hive2://localhost:10213 -e 'SHOW DATABASES;' &&
```